### PR TITLE
Run git clean after cloning repositories.

### PR DESCRIPTION
**Pull request by douglashall**
This cleans out any old artifacts like .pyc files.

Configuration Pull Request
---

Make sure that the following steps are done before merging:

  - [ ] A DevOps team member has approved the PR.
  - [ ] Are you adding any new default values that need to be overridden when this change goes live? If so:
    - [ ] Update the appropriate internal repo (be sure to update for all our environments)
    - [ ] If you are updating a secure value rather than an internal one, file a DEVOPS ticket with details.
    - [ ] Add an entry to the CHANGELOG.
  - [ ] If you are making a complicated change, have you performed the proper testing specified on the [Ops Ansible Testing Checklist](https://openedx.atlassian.net/wiki/display/EdxOps/Ops+Ansible+Testing+Checklist)?  Adding a new variable does not require the full list (although testing on a sandbox is a great idea to ensure it links with your downstream code changes).

### CODE REVIEW

**Review by feanil — APPROVED** [review]

File: playbooks/roles/git_clone/tasks/main.yml [hunk 1]
```diff
@@ -96,3 +96,10 @@
   tags:
     - install
     - install:code
+
+- name: Run git clean after checking out code
+  shell: cd {{ item.DESTINATION }} && git clean -xdf
```
Comment (douglashall) [inline playbooks/roles/git_clone/tasks/main.yml:101]: @feanil @fredsmith How do I refer to the git repo directory here? `item.DESTINATION` doesn't seem to work.

**Review by clintonb — APPROVED** [review]

**Comment by clintonb** [comment]
Nice! This solves the problem for every downstream play going forward!

**Comment by douglashall** [comment]
@fredsmith is very smart! I just know how to type.

File: playbooks/roles/git_clone/tasks/main.yml (hunk 1 above)
Comment (douglashall) [reply playbooks/roles/git_clone/tasks/main.yml:101]: Nevermind. I think I have it now.

**Review by fredsmith — APPROVED** [review]

**Pull request merged**